File: pset6/mario/more/mario.py
from cs50 import get_int


def get_positive_int(prompt):
    while True:
        n = get_int(prompt)
        if 0 < n < 9:
            break
    return n


def padding(g):
    for i in range(g):
        print(" ", end="")


def blocks(g):
    for i in range(g):
        print("#", end="")


Height = get_positive_int("Height: ")
Line = 1
for i in range(Height):
    padding(Height-Line)
    blocks(Line)
    print("  ", end="")
    blocks(Line)
    print()
    Line = Line + 1

File: pset1/mario/more/mario.c
#include <cs50.h>
#include <stdio.h>

void dash(int n)
{ for (int i = 0; i < n; i++)
    {printf("#");
    }
}    
void space(int n)
{ for (int i = 0; i < n; i++)
    {printf(" ");
    }
}    
int main(void)
    
{
    int x;
    int x2;
    int line;
      do
          {
          x = get_int("Height: \n");
          }
   while (x < 1 || x > 8);
        x2 = x;
        line = 1;
    while (x2>0)
        {
            space(x-line);
            dash(line-1);
            printf("#  #");
            dash(line-1);
            printf("\n");
         x2 = x2-1;
         line = line + 1;
        }     

}
